hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneClusterImpl.java

```diff
@@ -685,5 +756,23 @@ private void configureTrace() {
       }
       GenericTestUtils.setRootLogLevel(Level.INFO);
     }
+
+    private void configureRecon() throws IOException {
+      TemporaryFolder tempFolder = new TemporaryFolder();
+      tempFolder.create();
+      File tempNewFolder = tempFolder.newFolder();
+      conf.set(OZONE_RECON_DB_DIR,
+          tempNewFolder.getAbsolutePath());
+
+      File reconOmDbDir = tempFolder.newFolder();
+      conf.set(OZONE_RECON_OM_SNAPSHOT_DB_DIR, reconOmDbDir
+          .getAbsolutePath());
+
+      conf.set(OZONE_RECON_SQL_DB_JDBC_URL, "jdbc:sqlite:"+
+          tempNewFolder.getAbsolutePath()+"/ozone_recon_sqlite.db");
+
+      conf.set(RECON_OM_SNAPSHOT_TASK_INITIAL_DELAY_DEFAULT, "2s");
```

Comment:
Why are we setting a config value with the "DEFAULT" value constant as key? We should set RECON_OM_SNAPSHOT_TASK_INITIAL_DELAY and RECON_OM_SNAPSHOT_TASK_INTERVAL. Those are the config keys.